.github/workflows/tests.yml

```diff
@@ -230,13 +230,28 @@ jobs:
         go: ["1.22.x"]
         suite: ["cluster", "standalone"]
         backend: ["dir", "btrfs", "lvm", "zfs", "ceph", "random"]
+        event_type:
+          - ${{ github.event.name }}
         include:
           - go: stable
             suite: cluster
             backend: dir
           - go: stable
             suite: standalone
             backend: dir
+        exclude:
+          - event_type: pull_request
+            backend: lvm
+          - event_type: schedule
+            backend: lvm
+          - event_type: pull_request
+            backend: ceph
+          - event_type: schedule
+            backend: ceph
+          - event_type: pull_request
+            backend: random
+          - event_type: schedule
+            backend: random
```

Comment:
@tomponline @simondeziel I can't figure out why these exclude rules aren't working. (Also, I'm aware we only want to disable ceph and lvm on `push` events, I just wanted to test it works first in a PR)

Reply:
Try adding something like `if: github.event_name == 'pull_request'` on the workflow step itself

Reply:
you should be able to expand the logic to consult the matrix

Reply:
Ok thanks :)

Reply:
Turns out I was using `github.event.name` and not `github.event_name` :facepalm: